panther_lights/src/animation/battery_animation.py

```diff
@@ -6,16 +6,17 @@
 
 
 class BatteryAnimation(Animation):
-
     ANIMATION_NAME = 'battery_animation'
```

Comment:
I would keep that line.  It's just a matter of preference, but other animations have spacing  between class and ANIMAITON_NAME